components/splashPage.tsx

```diff
@@ -1,7 +1,8 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { Button } from '@nextui-org/react';
-import Link from 'next/link';
+import { Button, Pagination, Text } from '@nextui-org/react';
 import styles from '../styles/Home.module.css';
```

Comment:
Prefer consistency in where the style files are located. Currently this file is located in /styles, but all the other components you add have styles in /components

Reply:
:100: agree, I was just a bit lazy:P `.modules` is now the norm :)